torch_xla/csrc/runtime/BUILD

```diff
@@ -364,6 +364,7 @@ cc_library(
         "@org_tensorflow//tensorflow/compiler/xla/translate/hlo_to_mhlo:hlo_to_mlir_hlo",
         "@org_tensorflow//tensorflow/compiler/xla/translate/mhlo_to_hlo:mlir_hlo_to_hlo",
         "@org_tensorflow//tensorflow/compiler/xla/mlir_hlo:all_passes",
+        "@stablehlo//:stablehlo_serialization",
```

Comment:
@ManfeiBai FYI since this dependency is new.

Reply:
if [[stablehlo imported in TF](https://github.com/tensorflow/tensorflow/blob/1ec9f2e7163aff9c959ed0f478497ff475a99c16/third_party/stablehlo/workspace.bzl)] is the total same as [stablehlo imported in OpenXLA](https://github.com/openxla/xla/tree/main/third_party/stablehlo), this new dep `"@stablehlo//:stablehlo_serialization",` could be used after we migrate to pull XLA from OpenXLA, @lsy323 

my thought is this would be very like `tsl` because `tsl` is a third party included by OpenXLA too

cc @JackCaoG

Reply:
@ManfeiBai Good catch, you are right. It shouldn't be a problem when migrating to OpenXLA

Reply:
testing PR: https://github.com/pytorch/xla/pull/5237

Reply:
since build successed on tpu: https://pantheon.corp.google.com/cloud-build/builds;region=us-central1/46f7838f-af3f-4681-8f2e-04c5b46ea6e9?project=tpu-pytorch&e=13802955&jsmode=o&mods=allow_workbench_image_override

we could build with this new dep `@stablehlo//:...` after migrate to pull XLA from OpenXLA